packages/protocol/contracts/L1/TaikoErrors.sol

```diff
@@ -7,7 +7,8 @@
 pragma solidity ^0.8.18;
 
 abstract contract TaikoErrors {
-    // The following custom errors must match the definitions in other V1 libraries.
+    // The following custom errors must match the definitions in other V1
+    // libraries.
     error L1_ALREADY_PROVEN();
     error L1_BLOCK_ID();
     error L1_EVIDENCE_MISMATCH(bytes32 expected, bytes32 actual);
```

Comment:
in this file, the following `L1_INSUFFICIENT_TOKEN` error should also be removed? or we just want to keep this error?

Reply:
Done